amity-sample-code/build.gradle

```diff
@@ -37,6 +37,9 @@ android {
 
 dependencies {
     api project(path: ':amity-uikit')
+    // Added :chat module manually as it is unused in base amity-uikit module
+    implementation project(path: ':chat')
```

Comment:
[question]
Do we use it? Or, without it, are some build errors?

Reply:
We don't use it, but there are build errors and sometimes it affects the build on Jitpack. Some modules are compiled in parallel and if one of them fails, then the rest is cancelled.